### PR TITLE
new file:   packages/cpm/cpm.12.1.2/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
added dune-project file to remove upper-bound dune constraint

### CODE REVIEW

File: packages/cpm/cpm.12.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/cpmlib"
+bug-reports: "https://github.com/UnixJunkie/cpmlib/issues"
+dev-repo: "git+https://github.com/UnixJunkie/cpmlib.git"
+license: "LGPL-2.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/cpm/cpm.12.1.2/opam:10]:
```suggestion
  "dune" {>= "2.8"}
```

File: packages/cpm/cpm.12.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/cpmlib"
+bug-reports: "https://github.com/UnixJunkie/cpmlib/issues"
+dev-repo: "git+https://github.com/UnixJunkie/cpmlib.git"
+license: "LGPL-2.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "2.8"}
+  "batteries"
```
Comment (mseri) [inline packages/cpm/cpm.12.1.2/opam:11]:
```suggestion
  "batteries" {>= "2.4"}
```
Uses `BatUnix.run_and_read"

**Comment by UnixJunkie** [comment]
Thanks, I added the batteries lbound upstream.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**